spec/providers/atmosphere/local_pdp_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+describe Atmosphere::LocalPdp do
+  context '#can_manage?' do
+    let(:user) { build(:user, id: 1) }
```

Comment:
Please don't use `id` in build and than saving it in db (in line 16) - this can lead to hard to find bugs.